pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -84,6 +84,10 @@
     // Initial delays
     private static final String STATUS_CHECKER_INITIAL_DELAY_IN_SECONDS =
         "controller.statusChecker.initialDelayInSeconds";
+    private static final String RETENTION_MANAGER_INITIAL_DELAY_IN_SECONDS =
```

Comment:
It would be good to leave some comments to stagger these two tasks here.

Reply:
For the test I was pursuing, staggering didn't make a difference. I will however check this in to have the flexibility in the future.